### PR TITLE
Add icon button

### PR DESCRIPTION
**Pull request by MrF3lix**

### CODE REVIEW

File: src/components/external-provider-login.tsx [hunk 1]
```diff
@@ -2,6 +2,9 @@ import { Provider } from '@supabase/supabase-js';
 import { Capacitor } from '@capacitor/core';
 import { supabase } from '~/services/supabase-client';
 import { Button } from '~/components/button';
+import { GoogleIcon } from './icons/google-icon';
+import { FacebookIcon } from './icons/facebook-icon';
+import { AppleIcon } from './icons/apple-icon';
```
Comment (vkaelin) [inline src/components/external-provider-login.tsx:7]: I think the icons are missing or the paths are wrong.

**Review by vkaelin — CHANGES_REQUESTED** [review]
Icons missing

File: src/components/external-provider-login.tsx (hunk 1 above)
Comment (MrF3lix) [reply src/components/external-provider-login.tsx:7]: 🤦🏼‍♂️

Comment (MrF3lix) [reply src/components/external-provider-login.tsx:7]: I've added it

**Review by vkaelin — APPROVED** [review]
Great!

**Pull request merged**